roles/wildfly_systemd/tasks/apply_cp.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: "Execute only if eap_enable is true."
+  when:
+    - eap_enable is defined and (eap_enable | bool)
+  block:
+    - name: Check current EAP patch installed
```

Comment:
s/EAP/Wildfly ?

Reply:
Nop, this code is specific to EAP and there is no patch mecanism for Wildlfy.